src/main/java/duke/Pikachu.java

```diff
@@ -0,0 +1,227 @@
+            return;
+        }
+
+        /**
+         * "todo"
+         * Creates a todo task. 
+         * Syntax: todo <taskname>
+         */
+        if (split[0].toLowerCase().equals("todo")) {
+            String[] split2 = str.split(" ", 2);
+            //System.out.printf("For debugging. split2[1] = %s\n", split2[1]);
+            try {
+            ToDo t = new ToDo(split2[1]);
+            inputList.add(t);
+            System.out.println("Pikachu has added this task to the list!");
+            System.out.println("   > " + t);
+            System.out.println("You now have " + inputList.size() + " tasks in the list.");
+            return;
+            } catch (Exception e) {
+                System.out.println("Task description is empty!");
+                return;
+            }
+        }
+
+        /**
+         * "deadline"
+         * Creates a deadline task.
+         * Syntax: deadline <taskname> /<yyyy-mm-dd hhmm of deadline>
+         */
+        if (split[0].toLowerCase().equals("deadline")) {
+            try {
+            String[] split2 = str.split("/");
+            String[] split3 = split2[1].split(" ");
+            String[] split4 = split2[0].split(" ", 2);
+            String name = split4[1];
+            String date = split3[0];
+            String time = split3[1];
+            String[] yymmdd = date.split("-");
+            LocalDateTime deadline = LocalDateTime.of(Integer.parseInt(yymmdd[0]), Integer.parseInt(yymmdd[1]), 
+                    Integer.parseInt(yymmdd[2]), Integer.parseInt(time.substring(0,2)), Integer.parseInt(time.substring(2,4)));
+            //System.out.printf("For debugging. split2[1] = %s, split3[1] = %s\n", split2[1], split3[1]);
+            Deadline d = new Deadline(name, deadline);
+            inputList.add(d);
+            System.out.println("Pikachu has added this task to the list!");
+            System.out.println("   > " + d);
+            System.out.println("You now have " + inputList.size() + " tasks in the list.");
+            return;
+            } catch (Exception e) {
+                System.out.println("Task description is empty/Empty or invalid deadline timing has been specified!");
+                return;
+            }
+        }
+
+        /**
+         * "event"
+         * Creates an event task.
+         * Syntax: event <taskname> /<yyyy-mm-dd hhmm of start> <yymmdd hhmm of end>
+         */
+        if (split[0].toLowerCase().equals("event")) {
+            try {
+            String[] split2 = str.split("/");
+            String[] split3 = split2[1].split(" ");
+            String[] split4 = split2[0].split(" ", 2);
+            String name = split4[1];
+            String dateStart = split3[0];
+            String timeStart = split3[1];
+            String dateEnd = split3[2];
+            String timeEnd = split3[3];
+            String[] yymmddStart = dateStart.split("-");
+            String[] yymmddEnd = dateEnd.split("-");
```

Comment:
I like how you assigned the split[i] to their respective variables! It makes everything look neat and easy to understand!